pkg/app/web/package.json

```diff
@@ -11,8 +11,6 @@
     "test:coverage": "yarn test --coverage",
     "lint": "eslint --ext .ts,.tsx ./src",
     "lint:fix": "eslint --fix --ext .ts,.tsx ./src",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
```

Comment:
should we remove storybook-related dependencies as well? this file at line L22-24

Reply:
Nice catch!
Tbh, not sure why but the build does not pass when I remove them. So weird!
Let me deal with them in another PR.

Reply:
Sure 👍